### PR TITLE
Add a utility function for collecting modules under a given config key. NFC

### PR DESCRIPTION
**Pull request by nkcsgexi**

### CODE REVIEW

**Comment by nkcsgexi** [comment]
@swift-ci please test

File: Sources/SwiftDriver/Jobs/Planning.swift [hunk 1]
```diff
@@ -479,6 +479,13 @@ extension Driver {
     return []
   }
 
+  @_spi(Testing) public static func getModule(withConfigKey: String, _ configs: [AdopterConfig]) -> Set<String> {
```
Comment (artemcm) [inline Sources/SwiftDriver/Jobs/Planning.swift:482]: Given that this is part of Driver public interface, could we give this method a more-descriptive name please?
Something like `getAllConfiguredModules(with key: String, ...`.

**Review by artemcm — APPROVED** [review]

File: Sources/SwiftDriver/Jobs/Planning.swift (hunk 1 above)
Comment (nkcsgexi) [reply Sources/SwiftDriver/Jobs/Planning.swift]: Great point. Fixed.

**Comment by nkcsgexi** [comment]
@swift-ci please test

**Pull request merged**